scrapy/utils/gz.py

```diff
@@ -51,8 +51,12 @@ def gunzip(data):
     return output
 
 _is_gzipped_re = re.compile(br'^application/(x-)?gzip\b', re.I)
+_is_octetstream_re = re.compile(br'^(application|binary)/octet-stream\b', re.I)
```

Comment:
A small trick: it can be written as 

```py
_is_octetstream = re.compile(br'^(application|binary)/octet-stream\b', re.I).search
```

and then used like ` or (_is_octetstream(ctype) and cenc in ...)`